### PR TITLE
Added more top traded currencies and more data for intents

### PR DESCRIPTION
**Pull request by navendu-pottekkat**
Here are the things that I added:

- Added more top-traded currencies that were missing from the list

- Added some common messages that users give for show_accounts intent and transfer_money intent

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @erohmensing will take a look at it as soon as possible ✨

File: banking/nlu.md [hunk 1]
```diff
@@ -11,18 +11,31 @@
...
+- Singapore Dollar
 
 # intent:transfer_money <!--The user would like to transfer money.-->
 
 - I would like to make a transfer
 - I want to transfer money
+- I want to pay money
+- I want to pay someone
+- Can I transfer money?
+- pay my bills
```
Comment (erohmensing) [inline banking/nlu.md:27]: I think "pay bills" would probably be a different intent, usually it would lead to something else

**Review by erohmensing — CHANGES_REQUESTED** [review]
Thanks for the addition! All looks good except for the one comment I made :)

File: banking/nlu.md (hunk 1 above)
Comment (navendu-pottekkat) [reply banking/nlu.md]: I have removed that line. Now that I think about it, it should have been a different intent

**Comment by navendu-pottekkat** [comment]
@erohmensing Could you check now?

**Review by erohmensing — DISMISSED** [review]
Thanks for contributing!

**Comment by navendu-pottekkat** [comment]
> Thanks for contributing!

Happy to help!

**Review by EmmaWightman — APPROVED** [review]
This is awesome thank you @navendu-pottekkat

**Pull request merged**